src/plotSearchAndCompetitionsPage/components/mapPlotSearchOverlay.tsx

```diff
@@ -25,6 +27,24 @@ const usePreviousTarget = (value: SelectedTarget) => {
   return ref.current;
 };
 
+export const getMarkerIcon = (
+  symbol: string,
+  index: number,
+  isFavourite: boolean
+): DivIcon => {
+  const html = renderToStaticMarkup(
+    <div>
+      {isFavourite && <IconHeartFill className="favouriteIcon" />}
```

Comment:
I'd rather implement this inside `MapSymbol` with a boolean prop that triggers this to show up. And in that, the classname should also be prefixed with `MapSymbol` rather than being a totally loose class, i.e. `MapSymbol__favourite-icon`.